include/dem/pw_contact_force.h

```diff
@@ -80,6 +80,15 @@ class PWContactForce
     return torque_on_walls;
   }
 
+  /** This function is used to update the center of mass container member class
+   * @param Tensor would be added to the member class to make the update
+   */
```

Comment:
Revise English here.

Reply:
Just see would to will. Is there anything else @blaisb ?